examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -122,12 +117,26 @@ global  class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
                 (String) providerAndRate.get('serviceCode'),
                 (Decimal) providerAndRate.get('shipmentCost'),
                 (Decimal) providerAndRate.get('otherCost'),
-                (String) providerAndRate.get('serviceName')
+                (String) providerAndRate.get('serviceName'),
```

Comment:
should this be carrier?

Reply:
Yes .. Our Existing Heroku Service does not have any Carrier field. For this sample, we just populate carrier with the serviceName that we get from Heroku.